lib/commands/viewport.js

```diff
@@ -0,0 +1,36 @@
+import { imageUtil } from 'appium-support';
+
+let extensions = {}, commands = {};
+
+commands.getStatusBarHeight = async function () {
+  const {statusBar} = await this.uiautomator2.jwproxy.command(`/appium/device/system_bars`, 'GET', {});
+  return statusBar;
+};
+
+// this function is memoized in the constructor
+commands.getDevicePixelRatio = async function () {
```

Comment:
this call is actually not used, right?

Reply:
it is used during session initialization to populate the session capabilities